app/commands/deliver_to_subscriber.rb

```diff
@@ -0,0 +1,22 @@
+class DeliverToSubscriber
+  attr_reader :subscriber
+
+  def initialize(subscriber:)
+    @subscriber = subscriber
+    raise ArgumentError, "subscriber is required" if subscriber.nil?
+  end
+
+  def self.call(subscriber:)
+    DeliverToSubscriber.new(subscriber: subscriber).call
+  end
```

Comment:
I quite like writing these convenience methods like:

```ruby
def self.call(*args)
  new(*args).call
end
```

It means you don't have to update the method signature if you change `#initialize` or if you change the class name.

Reply:
Added in a75098d